wireless_modules/mqtt_client.py

```diff
@@ -1,8 +1,8 @@
 
 # Use micro pip to install packages
 import upip
-upip.install("micropython-umqtt.simple2")
-from umqtt.simple import MQTTClient
+upip.install("micropython-umqtt.robust2")
+from umqtt.robust import MQTTClient
```

Comment:
This isn't actually using `robust2` (https://github.com/fizista/micropython-umqtt.robust2), only `robust`! (https://github.com/micropython/micropython-lib/tree/master/umqtt.robust)

Reply:
That's because there actually isn't anything called umqtt.robust2. If you look at the previous changes, even with umqtt.simple2, we only imported from umqtt.simple.